plugins/generator-1.19.4/forge-1.19.4/templates/elementinits/entities.java.ftl

```diff
@@ -43,10 +43,10 @@ package ${package}.init;
 	<#list entities as entity>
 		<#if entity.getModElement().getTypeString() == "projectile">
 			public static final RegistryObject<EntityType<${entity.getModElement().getName()}Entity>> ${entity.getModElement().getRegistryNameUpper()} =
-				register("projectile_${entity.getModElement().getRegistryName()}", EntityType.Builder.<${entity.getModElement().getName()}Entity>
+				register("${entity.getModElement().getRegistryName()}", EntityType.Builder.<${entity.getModElement().getName()}Entity>
 						of(${entity.getModElement().getName()}Entity::new, MobCategory.MISC).setCustomClientFactory(${entity.getModElement().getName()}Entity::new)
 						.setShouldReceiveVelocityUpdates(true).setTrackingRange(64).setUpdateInterval(1).sized(0.5f, 0.5f));
-		<#else>
+		<#elseif entity.getModElement().getTypeString() == "livingentity">

```

Comment:
Keep this unrelated change that also is not needed out of the PR please

Reply:
Done